adapters/Unity/UnityAdapter/GADMAdapterUnity.m

```diff
@@ -60,7 +58,21 @@ - (void)stopBeingDelegate {
   if (_bannerAd != nil) {
     [_bannerAd stopBeingDelegate];
   }
-  [[GADMAdapterUnitySingleton sharedInstance] stopTrackingDelegate:self];
+}
+
+- (void)initializeWithGameID:(NSString *)gameID withInitDelegate:(id)initDelegate{
+  if (![UnityAds isSupported]) {
+    NSLog(@"Unity Ads cannot be initialized: this device is not supported.");
+  }
+  
+  if ([UnityAds isInitialized]) {
+    NSLog(@"Unity Ads has already been initialized.");
+  }
+  
+  // Metadata needed by Unity Ads SDK before initialization.
+  GADMUnityConfigureMediationService();
+  // Initializing Unity Ads with |gameID|.
+  [UnityAds initialize:gameID testMode:NO enablePerPlacementLoad:YES initializationDelegate:initDelegate];
```

Comment:
I think this could go into the `GADMUnityInitializationDelegate` class and that class could be the designated class used for initializing the UnityAds SDK and forwarding all necessary callbacks to the adapter.

Reply:
If I move the initialization code into the `GADMUnityInitializationDelegate` class, then the class is doing more than being a delegate.  Am I misunderstanding your comment, or maybe the `GADMUnityInitializationDelegate` could be renamed to something that is more appropriate for a change like this?

Reply:
I think it could me renamed similar to `GADMUnityInitializer` and it will take care of initializing the Unity Ads SDK across all the other adapters.
Or it can go into the Utils class ?